readme.md

```diff
@@ -54,7 +56,8 @@ avg.get() // 6, because (5 + 6 + 7) / 3
 
 ### `w.push(val)`
 
-`val` must be a number.
+`val` must be a number.    
+Return `this`.
```

Comment:
I think this phrasing is a bit confusing (this is my subjective judgement though). What do you think about "Returns `m`." ?

Reply:
Did you mean to reference the variable *w*, e.g., "Returns w." ?